src/controller/shareArticleController.js

```diff
@@ -0,0 +1,57 @@
+import dotenv from 'dotenv';
+import { helperMethods, sendEmail } from '../utilities';
+
+dotenv.config();
+
+/**
+ * Class representing the shareArticle controller
+ * @class ShareArticle
+ * @description controller used to handle routes for sharing articles
+ */
+class ShareArticle {
+/**
+  * Share an article
+  * Route: POST: /articles/share
+  * @param {object} req - Request object
+  * @param {object} res - Response object
+  * @return {res} res - Response object
+  * @memberof ShareArticle
+ */
+  static async ShareArticleViaEmail(req, res) {
+    const {
+      articleId,
+      title,
+      author,
+      email
+    } = req.body;
+    const articleURL = `${process.env.shareArticleURL}/${articleId}`;
```

Comment:
Nice one here, your code was quite easy to follow through. For the sake of memory management, variables declared from lines (21 - 27) might not be necessary since they were only used once in the `shareDetails ` object. Consider assigning the values directly in the object.

Reply:
Thank you for pointing this out. I have implemented your feedback. Thank you.